vector-test/src/test/scala/spec/geotrellis/vector/ExtentSpec.scala

```diff
@@ -95,6 +99,11 @@ class ExtentSpec extends FunSpec with Matchers {
       assert(e.contains(10.0, 10.0) === false)
     }
 
+    it("empty extent should contain empty extent") {
```

Comment:
Should empty extent contain an empty extent? How the contain function should behave on extent bounds?